Create Report Directory and Add First Two Section Files (#24)
ConorMFinn/CO600-SeriousCyberSec: report/2_background.txt

Existing Solutions
Gamification has been used in education to great effect in the past. One such example is the study done to the Vienna University of Technology (Adrian Dabrowski, 2014). In this paper, the entire course structure was adapted into a gamified version. In addition to ordinary lectures that would still be attended, students were expected to complete a range of “challenges”. These challenges were designed to fit into a “movie style” hacking environment to make the experience of completing the challenges more entertaining, and potentially easier to remember. The study also took advantage of the progress reporting and competition gamification techniques by providing a leaderboard for students on the course. The study also made use of achievements and badges, also a well-known gamification technique, to motivate students to do well. The study found that 96% of the students that took the course enjoyed the “gaming-like aspects” of the practical exercise. 57% also responded that gamification of topics would help to encourage students to take cyber security modules and courses. 

> Another study at Leeds Beckett University (Z. Cliffe Schreuders, 2016) which used gamification as a method to improve motivation for completing learning tasks beyond assignments. The methodology involved the creation of three kinds of experience points (XP), with each focusing on a different element of the course: practical skills, knowledge, and reflection. Tasks to be completed were given XP values and were framed as “quests”. Students were also given access to an online tool that allowed them to track the amount of each XP they had accrued at the year progressed. All students responded that they liked having access to their progress at any time, with 63% liking seeing their relative XP compared to other students. All students taking part in the course also said that they did more out of class work than they would have done otherwise. 

Author: ```suggestion
Another study at Leeds Beckett University (Z. Cliffe Schreuders, 2016) which used gamification as a method to improve motivation for completing learning tasks beyond assignments. The methodology involved the creation of three kinds of experience points (XP), with each focusing on a different element of the course: practical skills, knowledge, and reflection. Tasks to be completed were given XP values and were framed as “quests”. Students were also given access to an online tool that allowed them to track the amount of each XP they had accrued at the year progressed. All students responded that they liked having access to their progress at any time, with 63% liking seeing their relative XP compared to other students. All students taking part in the course also said that they did more out-of-class work than they would have done otherwise. 
```

> Research conducted at the Delft University of Technology (Iosup, 2014) involved the use of a points system to motivate students to complete additional activities. Grades within the course are determined based on the number of points achieved while the course is running. The results show a 10% increase in the number of students that completed the course and  found that 50-75% of the students in each class felt better motivated by gamification methods. 

Author: ```suggestion
Research conducted at the Delft University of Technology (Iosup, 2014) involved the use of a points system to motivate students to complete additional activities. Grades within the course were determined based on the number of points achieved while the course was running. The results showed a 10% increase in the number of students that completed the course and  found that 50-75% of the students in each class felt better motivated by gamification methods. 
```
